torch/linalg/__init__.py

```diff
@@ -1480,6 +1480,22 @@
   Please use :func:`torch.linalg.svdvals`, which computes only the singular values,
   instead of `compute_uv=False`.
 
+.. note:: Selection of `driver` (this keyword argument only works on CUDA inputs with cuSOLVER backend):
+
+    - When `driver` is not given or set to `None` (*default*), a heuristic of drivers will be selected based on
+      matrix sizes and batch sizes of the inputs.
+      The default choice should be efficient and precise for most inputs. However, if input matrices
+      have large condition numbers or could be ill-conditioned, it's recommended to choose the `gesvd` driver.
+    - The `gesvd` driver directly calls the cuSOLVER `cusolverDn<t>gesvd` method. This QR-based algorithm is
+      less performant on GPU, but it can converge ill-conditioned matrices and get high precision for those inputs.
+    - The `gesvdj` driver directly calls the cuSOLVER `cusolverDn<t>gesvdj` method. This Jacobi-based algorithm
+      has better performance than `gesvd` on GPU, but it may get less accurate results for ill-conditioned matrices or
+      matrices with very large sizes (be cautious if matrix size > 2048).
+    - The `gesvdjBatched` driver directly calls the cuSOLVER `cusolverDn<t>gesvdjBatched` method. It only works for
+      batched square matrices with size less than or equal to 32.
+    - The `gesvdaBatched` driver directly calls the cuSOLVER `cusolverDn<t>gesvdaStridedBatch` method. This is an
```

Comment:
Is this driver only callable on batches? Maybe for user clarify we should rename it to `gesvda` if it can be called on two matrices or batches of matrices?

Reply:
`gesvda` driver will work on single matrix as well. I just removed the "Batched" in driver string name and changed it to `gesvda`.